apps/src/lib/levelbuilder/lesson-editor/LevelTokenDetails.story.jsx

```diff
@@ -4,89 +4,57 @@ import {action} from '@storybook/addon-actions';
 
 const defaultLevel = {
   id: '10',
+  position: 1,
   levels: [
     {
-      name: 'Level One',
-      id: '1',
-      url: 'levels/598/edit',
-      icon: 'fa-desktop',
-      isUnplugged: false,
-      isConceptLevel: true,
-      skin: null,
-      videoKey: null,
-      concepts: '',
-      conceptDifficulty: ''
+      name: 'Level 1',
+      id: '2',
+      url: '/fake/url/'
     }
   ],
-  position: 1,
-  activeId: '1',
-  kind: 'puzzle',
-  bonus: false,
-  assessment: false,
-  challenge: false,
-  expand: false
+  activeId: '2',
+  expand: true
 };
 
-const blocklyLevel = {
-  id: '11',
-  position: 1,
-  levels: [
-    {
-      id: '4',
-      name: 'blockly:Studio:playlab_1',
-      url: 'levels/59800/edit'
-    }
-  ],
-  activeId: '4',
-  expand: false
+export default {
+  title: 'LevelTokenDetails',
+  component: LevelTokenDetails
 };
 
-export default storybook => {
-  storybook.storiesOf('LevelTokenDetails', module).addStoryTable([
-    {
-      name: 'level token details',
-      story: () => (
-        <div style={{width: 800}}>
-          <LevelTokenDetails
-            setScriptLevelField={action('setScriptLevelField')}
-            scriptLevel={defaultLevel}
-            activitySectionPosition={1}
-            activityPosition={1}
-            lessonExtrasAvailableForUnit={true}
-            allowMajorCurriculumChanges={true}
-          />
-        </div>
-      )
-    },
-    {
-      name: 'level token details with bonus disabled',
-      story: () => (
-        <div style={{width: 800}}>
-          <LevelTokenDetails
-            setScriptLevelField={action('setScriptLevelField')}
-            scriptLevel={defaultLevel}
-            activitySectionPosition={1}
-            activityPosition={1}
-            lessonExtrasAvailableForUnit={false}
-            allowMajorCurriculumChanges={true}
-          />
-        </div>
-      )
-    },
-    {
-      name: 'level token details with blockly level',
-      story: () => (
-        <div style={{width: 800}}>
-          <LevelTokenDetails
-            setScriptLevelField={action('setScriptLevelField')}
-            scriptLevel={blocklyLevel}
-            activitySectionPosition={1}
-            activityPosition={1}
-            lessonExtrasAvailableForUnit={true}
-            allowMajorCurriculumChanges={true}
-          />
-        </div>
-      )
-    }
-  ]);
+const Template = args => (
+  <div style={{width: 800}}>
```

Comment:
I know this was here before but do you know why we need this?

Reply:
Good Q! I just tried without it, and it keeps the display from wrapping, and from filling the whole width of the screen. With the width defined, when I shrink the screen size it truncates instead of wrapping. I imagine this is intentional!